test/integration/hotrestart_test.sh

```diff
@@ -272,8 +283,30 @@ do
   run_testsuite "$HOT_RESTART_JSON" "1" || exit 1
 done
 
+# Hotrestart in specified UDS
+# Run one of the tests with real symbol tables. No need to do all of them.
```

Comment:
real symbol tables are the default, so I'd run just one with fake symbol tables (switch the "0" and "1" in the second arg in the two run_testsuite calls below).

Reply:
Done.